extensions/wikia/MercuryApi/MercuryApiHelper.class.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class MercuryApiHelper {
+
+	/**
+	 * Check if a string is a valid article title
+	 *
+	 * @param string $title
+	 * @return bool
+	 */
+	public static function isTitleStringValid( $title ) {
+		return !is_null( $title ) && trim( $title ) !== '';
```

Comment:
Guys, I'm wondering if we should actually do the trimming. Shouldn't we let the MW determine if the just whitespace article title is OK or not? We would get this info from here: https://github.com/Wikia/app/blob/CONCF-432/extensions/wikia/MercuryApi/MercuryApiController.class.php#L170 as the title string is taken care of here https://github.com/Wikia/app/blob/dev/includes/Title.php#L146.

On the other hand, instead of using trim, we could check if the title string consists only of whitespace characters **and underscores**, to return the same error code as MW.
http://elderscrolls.wikia.com/wiki/%20_ → 400 bad request
http://elderscrolls.wikia.com/wikia.php?controller=MercuryApi&method=getArticle&title=%20_ → 404 not found

Or maybe it should be handled in some other way?